utils/extension.ts

```diff
@@ -27,7 +27,7 @@ export const getAddress = async (address: string) => {
 
 export const getSelectedAccount = (accounts: WalletAccount[]) => {
   try {
-    const selectedAddress = localStorage.getItem('kodaauth')
+    const selectedAddress = useWalletStore().selected?.address
     const account = accounts.find(
       (account) => account.address === selectedAddress,
     )
```

Comment:
Let's remove it as it's never been used.

Reply:
done